test/Interop/Cxx/static/static-member-var.swift

```diff
@@ -0,0 +1,74 @@
+// Test that static member variables are handled properly by the ClangImporter.
```

Comment:
I'm not a huge fan of comments that repeat other information. "Test that ${test name with spaces instead of dashes} are handles properly" -- that sounds like a description every test ever written. I'd rather have no comment at all rather than a repetitive boilerplate comment. Of course, feel free to add a meaningful comment, if there is anything to say. (In all files in this PR.)